### DISCUSSION TITLE
Software Installations

### DISCUSSION THREAD

**Discussion by akash-coded** [akash-coded/mern#4] [Guidelines]
1. [VSCode](https://code.visualstudio.com/download)
2. [Git SCM](https://git-scm.com/downloads)

**Comment by Sridhar-Kesari** [comment]
Done

**Comment by PriyankaSejwal** [comment]
done

**Comment by Priyavs-coder** [comment]
done

**Comment by SKV5699** [comment]
> 1. [VSCode](https://code.visualstudio.com/download)
> 2. [Git SCM](https://git-scm.com/downloads)

installed
